plugins/content/joomla/src/Extension/Joomla.php

```diff
@@ -179,9 +179,15 @@ public function onContentAfterSave(AfterSaveEvent $event): void
                     'name'     => $user->name,
                     'email'    => PunycodeHelper::emailToPunycode($user->email),
                     'title'    => $article->title,
-                    'url'      => Route::link('administrator', 'index.php?option=com_content&view=articles&filter[search]=id:' . $article->id, false, $linkMode, true),
                 ];
 
+                if ($receiver->authorise('core.login.admin')) {
```

Comment:
```suggestion
                $receiver->authorise('core.login.admin') && $receiver->authorise('core.manage', 'com_content')
```
We also need to check to make sure user has core.manage permission to be allowed to access to Articles management screen.